Makefile

```diff
@@ -0,0 +1,12 @@
+# Help Helper matches comments at the start of the task block so make help gives users information about each task
+.PHONY: help
+help: ## Displays information about available make tasks
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+build-clusters: ## Build all clusters in parallel.
```

Comment:
@DanHartman this can spin up 4x clusters in (2 in AWS, 2 in GCP) in about 30 minutes. Thanks for putting me on to parallel. Getting it working with terraform so you can still ctrl+c out was uh... annoying.